src/main/resources/templates/test.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
What is this used for?

Reply:
was to test controller I'll remove this too